### ISSUE TITLE
Magicento 2 Indexer returns NULL throws IllegalStateException (v0.3.0)

### ISSUE THREAD

**Issue by OLTC-fperrin** [enriquepiatti/Magicento#184]
Hello,

To my pleasant surprised I noticed the update for Magicento2 but since I've installed the new version I can't use it as it throws 100+ Exception while indexing, here is what I get in the Event Log:

```
10:00:32 IllegalStateException
         Error while indexing F:\www\magento2\app\etc\env.php
         To reindex this file IDEA has to be restarted: @NotNull method com/magicento2/extensions/index/Magicento2DataIndexerAbstract.map must not return null
10:00:35 IllegalStateException
         Error while indexing F:\www\magento2\dev\tests\functional\tests\app\Magento\Cms\Test\Constraint\AssertCmsBlockInGrid.php
         To reindex this file IDEA has to be restarted: @NotNull method com/magicento2/extensions/index/Magicento2DataIndexerAbstract.map must not return null
[...]
10:01:08 IllegalStateException
         Error while indexing F:\www\magento2\vendor\magento\framework\DataObject\Copy\Config\Data\Proxy.php
         To reindex this file IDEA has to be restarted: @NotNull method com/magicento2/extensions/index/Magicento2DataIndexerAbstract.map must not return null
10:01:08 com.intellij.diagnostic.MessagePool$TooManyErrorsException: Too many IDE fatal errors. Monitoring stopped.
10:01:08 com.intellij.diagnostic.MessagePool$TooManyErrorsException: Too many IDE fatal errors. Monitoring stopped.
[...]
```

I spared you a _few_ lines

Hope to see a fix soon

Thank you for your work

**Comment by kevinjavitz** [comment]
Same for me, prevents using Magicento2 at all :( I can't even enable it. When I enable it says enabled. But it is not running. I press alt + m to enable again, still nothing.

**Comment by enriquepiatti** [comment]
Try downloading magicento2 again, it should be fixed in 0.3.1

**Comment by OLTC-fperrin** [comment]
Yep it's fixed! Great news!

Thank you
